sdk/nodejs/output.ts

```diff
@@ -319,6 +323,39 @@ function copyResources(resources: Set<Resource> | Resource[] | Resource) {
     return copy;
 }
 
+async function liftInnerOutput(allResources: Set<Resource>, value: any, isKnown: boolean, isSecret: boolean) {
```

Comment:
I know this isn't exposed publicly - but it would be *very* useful to have a comment here that made clear internally what the contract was for this function.  I've read a few times, and I'm still not exactly sure I understand what it's contract is.

The "Note" below partly explains - but I think it at the very least needs to be re-written to make sense given new code organization.

Reply:
Yes - the note you call out - and I think you are right that it is correct modulo the "transformed value" part.